client/lcd/test_helpers.go

```diff
@@ -211,9 +211,8 @@ func (b AddrSeedSlice) Swap(i, j int) {
 // their respective sockets where nValidators is the total number of validators
 // and initAddrs are the accounts to initialize with some steak tokens. It
 // returns a cleanup function, a set of validator public keys, and a port.
-func InitializeTestLCD(
-	t *testing.T, nValidators int, initAddrs []sdk.AccAddress,
-) (cleanup func(), valConsPubKeys []crypto.PubKey, valOperAddrs []sdk.ValAddress, port string) {
+func InitializeTestLCD(t *testing.T, nValidators int, initAddrs []sdk.AccAddress) (
```

Comment:
☹️ really don't think this change is necessary

Reply:
sry - we need to have a formatting convention

Reply:
hmm yeah fair enough -- developer preference here 👍  (Although I'd never agree to such a convention as this)